Added Intro, removed TODO (#14723)
dotnet/docs: docs/architecture/grpc-for-wcf-developers/index.md

## Introduction

> Helping machines communicate with each other has been one of the primary preoccupations of the digital age. 

Reviewer: Maybe a simpler language?

```suggestion
Helping machines communicate with each other has been one of the main concerns of the digital age. 
```

## Purpose

> The popular framework 'gRPC' is a natural choice for new apps and WCF apps moving to .NET Core.  This is a short introduction to building gRPC Services in ASP.NET Core 3.0 and offers guidance for migrating an existing WCF application to gRPC.

Reviewer: ```suggestion
The popular framework gRPC is a natural choice for new apps and existing WCF apps moving to .NET Core.  This book is a short introduction to building gRPC services in ASP.NET Core and offers guidance for migrating an existing WCF application to gRPC.
```

## Who should use this guide

> [//]: #(UPDATE THIS)

Reviewer: Should we just remove this?

```suggestion
```

Reviewer: Still unclear why these are still here.

> 

Reviewer: ```suggestion
```

The audience for this guide is WCF developers, development leads, and architects who are interested in migrating WCF solutions on .NET 4 and earlier to ASP.NET Core 3.0 using gRPC services.

## How you can use this guide

> [//]: #(UPDATE THIS)

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```
